api/citiesModels/cities.js

```diff
@@ -0,0 +1,14 @@
+const Parse = require('parse/node')
+
+
+// Get current user
+class Cities {
```

Comment:
rename this class to just `User` since this will probably be used outside of just the cities routes (like in friends etc..)

Reply:
Adding this to next PR!